README.md

```diff
@@ -1,221 +1,172 @@
 
 ### 🎛 Pipe options
 
-You can select a pipe by clicking it on the canvas. The options of the pipe can than be changed at the bottom of the screen. There are a number of tabs splitting the options in a organized manner.
+You can select an element by double clicking it on the canvas. The options of the pipe will be opened in a modal in the middle of the screen. Information about the element is given at the top of the modal and it also has a button that will take you to the Frank!Doc of the element. In the middle section it is possible to add, edit and delete attributes on the element.  Finally there is a delete button at the bottom, which will remove the element all together.
 
 ### ⌨ Editor
 
-While the flow can help you manipulate your adapter, there are still some people that want some more control or are used to editing the adapter in XML-code.
+While the Flow can help you manipulate your adapter, there are still some people that want some more control or are used to editing the adapter in XML-code. The Editor had a lot of features similar to Visual Studio Code because it is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/)
 
-The editor can be opened by right-clicking the canvas and selecting “toggle editor”.
+### 🔁 Hybrid
 
-The editor has some special features:
+The Hybrid mode is a combination of the Editor and the Flow modes. It will show you every modification you make in real-time. For example, if you add an element to the Flow, it will automatically get inserted into the XML and the other way around. This mode is a great way to learn XML or to get an overview of an already existing configuration. 
 
-- The editor has autocompletion for the pipes.
-- The editor can highlight the selected pipe on the canvas.
-- Errors will be shown with red lines and messages in the editor.
+### ⚙️ Configuration Flow Settings
 
-#### Autocompletion
+One of the important things that may get overlooked, is the shareability of the configurations. After creating a configuration in the Flow, it will have a specific layout that would (hopefully) look pleasing. To ensure that others see the Flow the same way as you do, it could be useful to add your Flow settings to the configuration. This can be done by placing them as attributes in the  `configuration` tag of your configuration.
 
-The XML autocompletion is custom made for the Frank!Flow and is made available for everybody that wants XML completion based on XSD’s for the Monaco Editor. Like this project, it is open-source and can be found [here on Github](https://github.com/philipsens/monaco-xsd-code-completion).
+The available settings are:
 
-#### Validation
+| Attribute           | Available values                  |
+| ------------------- | --------------------------------- |
+| `flow:direction`    | `bezier`, `flowchart`, `straight` |
+| `flow:forwardStyle` | `horizontal`, `vertical`          |
+| `flow:gridSize`     | `0`, `10`, `25`, `50`, `100`      |
 
```

Comment:
How to edit the &lt;Configuration&gt; tag? Can you do it in Flow mode or only in the Editor? And how to do it in the Flow mode?

Reply:
For now it's just in the editor, and by mentioning that the tag has to be edited, I thought that that was pretty clear.

But you made me think. In the future I will possibly add a feature to do this in a graphical manner. I'll then also edit the readme.

Reply:
Ok.